nshlib/nsh_telnetlogin.c

```diff
@@ -161,9 +162,19 @@ static void nsh_telnettoken(FAR struct console_stdio_s *pstate,
 int nsh_telnetlogin(FAR struct console_stdio_s *pstate)
 {
   char username[16];
-  char password[16];
+  char password[128];
+#ifdef CONFIG_NSH_PLATFORM_CHALLENGE
```

Comment:
can second commit be moved to a separate PR?
It is a bit out of context of this PR

Reply:
I see... It is already done

Reply:
Sorry, I append the patch to the wrong PR. that patch should belong https://github.com/apache/nuttx-apps/pull/1459.